operations/billingautomations.md

```diff
@@ -0,0 +1,707 @@
+| `Prepayment` | [Billing automation prepayment type](billingautomations.md#billing-automation-prepayment-type) | required | Type of prepayment. |
+| `AssignmentTargetType` | [Billing automation assignment target type](billingautomations.md#billing-automation-assignment-target-type) | required | Type of target company and customer assignment behavior. |
+| `TriggerType` | [Billing automation trigger type](billingautomations.md#billing-automation-trigger-type) | required | Trigger type of billing automation. |
+| `BillAggregationType` | [Billing automation bill aggregation type](billingautomations.md#billing-automation-bill-aggregation-type) | required | Type of bill aggregation. |
+| `CreatedUtc` | string | required | Creation date and time of the billing automation in UTC timezone in ISO 8601 format. |
+| `ProcessingStartOffset` | string | optional | Processing start offset. Applicable only if the billing automation `TriggerType` is set to recurring. |
+| `OrderItemConsumptionPeriod` | [Billing automation order item consumption period type](billingautomations.md#billing-automation-order-item-consumption-period-type) | optional | Specifies the type of order item consumption period for the billing automation. This value is applicable only when the `TriggerType` is set to recurring. |
+| `Assignments` | array of [Billing automation assignment](billingautomations.md#billing-automation-assignment) | optional, max 20 items | List of billing automation assignments. |
+
+#### Company with billing automation relation
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `CompanyId` | string | required | Unique identifier of the company. |
+| `CompanyRelations` | [Company with relations parameters](billingautomations.md#company-with-relations-parameters) | required | Company relations. |
+
+#### Company with relations parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `PartnerCompany` | boolean | required | Indicates whether the billing automation will apply if the company added to a reservation is a partner company. |
+| `TravelAgency` | boolean | required | Indicates whether the billing automation will apply if the company added to a reservation is a travel agency. |
+
+#### Billing automation prepayment type
+
+* `All` - All
+* `Prepaid` - Prepaid
+
+#### Billing automation assignment target type
+
+* `CompanyAsDetails` - The company will be the associated account in the resulting routed bill, and the customer is the owner.
+* `CompanyAsOwner` - The company will be the owner in the resulting routed bill, and the customer might be the associated account.
+* `NoCompany` - The customer will be owner of the bill.
+
+#### Billing automation trigger type
+
+* `Continuous` - Billing automation is continuously triggered by reservation actions, such as creation, updates, and rebates.
+* `Recurring` - Billing automation is executed monthly.
+* `Legacy` - Legacy
```

Comment:
thought: this will be on domain team, maybe explanation what Legacy means, I think just stating this will be confusing for partners to what to do with it.

Reply:
[#68559](https://github.com/MewsSystems/mews/pull/68559) Coming.